Snowdog_Components/components/Templates/paypal-review/paypal-review.config.js

```diff
@@ -1,278 +1,288 @@
 module.exports = {
```

Comment:
single quotes in this file were correct. we have eslint config in `Snowdog_Components` and eslint extension (worth installing if you don't have it already) doesn't like this change 
![image](https://user-images.githubusercontent.com/49198312/129048435-8425216d-2bcf-45ed-aefd-006cd090f16b.png)